docs/about_yab/how_it_works.md

```diff
@@ -0,0 +1,62 @@
+# How it works?
+
+How can we offer a system where the users don't have to trust a facilitator to exchange their assets from an L2 to
+Ethereum?
+
+We propose a simple protocol that follows these steps:
+
+1. The user specifies a destination address on Ethereum and locks X tokens
+   to be bridged into an L2 escrow smart contract.
+
+2. The market maker is monitoring for a change of state in the escrow smart contract.
+
+3.
+
+a. The market maker calls the transfer function of the PaymentRegistry Contract in Ethereum.
+
+b. The transfer function of the PaymentRegistry contract in Ethereum pays the X tokens to the user.
+
```

Comment:
The correct layout isn't displayed well in the site. The blog post in yetanotherbridge had the same issue and I opted for changing it to 3. and 4. instead of 3. a & b